datafusion-cli/src/exec.rs

```diff
@@ -340,14 +340,6 @@ mod tests {
         let session_token = "fake_session_token";
         let location = "s3://bucket/path/file.parquet";
 
-        // Missing region
```

Comment:
https://github.com/apache/arrow-rs/pull/5244

Reply:
maybe we can keep the test validating that there is no error if the region is not specified

Reply:
Done in 5a6d80d93